Makefile

```diff
@@ -22,7 +22,11 @@ setup:
 build: build-web build-go
 
 build-go:
-	go build ./...
+	# Ignore mmal.
```

Comment:
I see that `mmal` ignore is carried over from gostream build system; but do you have any insight on why we are excluding?

Reply:
I don't think we are using it anymore, and when we were, we didn't want to have to build it for HW that didn't need it, if I had to guess. I created https://viam.atlassian.net/browse/RSDK-5528 to track removing it altogether. I wouldn't want to do it here since moving + refactoring would be painful.